hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/amrmproxy/AMRMProxyService.java

```diff
@@ -779,7 +792,7 @@ private RequestInterceptorChainWrapper getInterceptorChain(
           .getApplicationId())) {
         throw new YarnException(
             "The AM request processing pipeline is not initialized for app: "
-                + appAttemptId.getApplicationId().toString());
+            + appAttemptId.getApplicationId().toString());
```

Comment:
Do we need toString()?

Reply:
toString() should be removed, I will fix it.